packages/rulesets/generated/spectral/az-arm.js

```diff
@@ -1484,6 +1484,21 @@ const longRunningResponseStatusCodeArm = (methodOp, _opts, ctx) => {
     return longRunningResponseStatusCode(methodOp, _opts, ctx, validResponseCodesList);
 };
 
+const getCollectionOnlyHasValueAndNextLink = (properties, _opts, ctx) => {
```

Comment:
@bdefoy I got the changes in `az-*.js` files by running `rush build`. Is it possible you forgot to run this as part of one of the already-merged PRs you did?

Reply:
Per my chat with @bdefoy, it is possible that `rush build` was missing. As a side-note, `rush regen-ruleindex` is used to update `rules.md`.